projects/packages/sync/changelog/fix-extend-sync-multisite-callables

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: changed
+Comment: jetpack Sync :: Migrate Jetpack multisite callables into the plugin vs defining them in the sync package. No functional change.
+
+
```

Comment:
I'll leave it up to you, but so far for packages I've included every change (versus the "only user-facing changes" policy we follow for the Jetpack plugin).

Reply:
Yeah I'm on the fence with the packages. Sync isn't usable without Jetpack  at this time so the changelog entry at the package level isn't needed. 

I think when we get it fully separated then each change might be warranted so consumers are aware of all changes.